tests/sentry/options/test_manager.py

```diff
@@ -258,6 +259,22 @@ def test_flag_prioritize_disk(self):
         with self.settings(SENTRY_OPTIONS={"prioritize_disk": None}):
             assert self.manager.get("prioritize_disk") == "foo"
 
+    def test_flag_prioritize_disk_falsy(self):
+        self.manager.register(
+            "prioritize_disk_falsy",
+            default=1,
+            flags=FLAG_PRIORITIZE_DISK | FLAG_AUTOMATOR_MODIFIABLE,
+        )
+        assert self.manager.get("prioritize_disk_falsy") == 1
+        assert self.manager.can_update("prioritize_disk_falsy", 0, UpdateChannel.AUTOMATOR) is None
+
+        with self.settings(SENTRY_OPTIONS={"prioritize_disk_falsy": 0}):
+            assert self.manager.get("prioritize_disk_falsy") == 0
+            assert (
+                self.manager.can_update("prioritize_disk_falsy", 0, UpdateChannel.AUTOMATOR)
+                == NotWritableReason.OPTION_ON_DISK
+            )
```

Comment:
This will mean that options that could previously be modified at runtime will cease to be mutable. Perhaps that is a benefit though as we shouldn't be changing 'disk' options at runtime.

Reply:
yes, this is intended, since it has `FLAG_PRIORITIZE_DISK`.

Reply:
As long as there is a value on disk, the option will be immutable.